features/junit_format.feature

```diff
@@ -538,6 +538,7 @@
       """
     And the file "junit/default.xml" should be a valid document according to "junit.xsd"
 
+  @php-version @php5.3 @php5.4
```

Comment:
no need to put anything there, as 5.3 is the min version supported by Behat instead. And using it to exclude PHP 7 is wrong though. It has this behavior only because the code used on Travis actually treats PHP 7.0 as a 5.0 release (so excluding any test with a min version tag)